demo/src/test/java/com/example/test/pages/KBBMOOCProject/ButtonSpecsvsFeatures.java

```diff
@@ -0,0 +1,21 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ButtonSpecsvsFeatures extends KbbBasePage {
+
+    public ButtonSpecsvsFeatures(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+    WebElement btnSpecsvsFeatures = keyword.findWebElementByXpath("//div[@label='Specs & Features']");
```

Comment:
should write the web element locator in separate line then call into a method/function